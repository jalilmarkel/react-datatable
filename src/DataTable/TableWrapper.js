import styled from 'styled-components';

const TableWrapper = styled.div`
  display: table;
  position: relative;
  height: 100%;
  width: 100%;
`;

export default TableWrapper;
