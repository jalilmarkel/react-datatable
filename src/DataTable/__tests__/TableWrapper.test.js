import 'jest-styled-components';
import React from 'react';
import { render, cleanup } from 'react-testing-library';
import TableWrapper from '../TableWrapper';

afterEach(cleanup);

test('should render correctly', () => {
  const { container } = render(<TableWrapper />);
  expect(container.firstChild).toMatchSnapshot();
});
