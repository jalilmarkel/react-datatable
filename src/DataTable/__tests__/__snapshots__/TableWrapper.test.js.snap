// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component <TableWrapper /> should render correctly 1`] = `
.c0 {
  display: table;
  position: relative;
  height: 100%;
  width: 100%;
}

<div
  class="c0"
/>
`;
